### DISCUSSION TITLE
Full Node menu is not loading

### DISCUSSION THREAD

**Discussion by Orest-rt** [Chia-Network/chia-blockchain#5730] [Support]
Hello! 

I have a problem for several days. When i start Chia client i can see just loading circles in Full Node. Farming doesn't working and writes "wait for sync". How can i fix it? 

Thank you! 

PC: Win10, Core i5 10400, 16 GB RAM

![123](https://user-images.githubusercontent.com/84587871/119171031-b4f32500-ba6c-11eb-91e3-e02e38f65ec7.png)

**Comment by BrandtH22** [comment]
@Orest-rt , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
